lib/maxmind/db.rb

```diff
@@ -153,8 +153,7 @@ def get(ip_address)
     # If no record is found for the IP address, the record will be +nil+ and
     # the prefix length will be the value for the missing network.
     #
-    # @param ip_address [String] a string in the standard notation. It may be
-    #   IPv4 or IPv6.
+    # @param ip_address [String, IPAddr] IPv4 or IPv6 address.
```

Comment:
Could you update the docs for `get` as well?

Reply:
Updated.